data/org.freedesktop.portal.WebExtensions.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (C) 2022 Canonical Ltd
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+      org.freedesktop.portal.WebExtensions:
+      @short_description: WebExtensions portal
+
+      The WebExtensions portal allows sandboxed web browsers to start
+      native messaging servers installed on the host system.
+
+      This documentation describes version 1 of this interface.
+  -->
```

Comment:
I think I miss a bit more clarity in this first documentation. Concretely, I miss:

 * Who is expected to use (and, perhaps more importantly, who is expected not to use) this portal
 * Somehow, a link to the relevant web-extensions documentation

This portal is not specific to web browsers anymore, so that bit can be removed. @jadahl @matthiasclasen if you have better suggestions for documentation, please share.

Reply:
Olivier shared his first draft of the documentation at https://phabricator.services.mozilla.com/D140803?id=659289#diff-change-9HfSkEzHsXuz

I added my first review of the documentation at https://phabricator.services.mozilla.com/D140803#5473608.

Please chime in there if you think that you have something to add :)